src/generator-vulcanjs/generators/start/index.js

```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+const VulcanGenerator = require('../../lib/VulcanGenerator');
+
+module.exports = class extends VulcanGenerator {
+
+  default() {
+    this.log(chalk.green('\nStarting your app... \n'));
+    this.spawnCommandSync('meteor', ['run']);
```

Comment:
Hmm im not convinced. What's your meteor version? 

My reasoning is that if we're doing a replacement for "mereor run", then the term should be "vulcan run".  But we're doing "vulcan start", so it should replace "meteor start". 

What do yoı think,